### ISSUE TITLE
[bug] Mixin Issue (1.17.1) (With Logs; No Other Mods)

### ISSUE THREAD

**Issue by mattfbacon** [Siphalor/mouse-wheelie#129]
**Version**
1.17.1. Mod versions in the log.
**Describe the bug**
The game crashes with an error message about mixins.

**To Reproduce**
1. Launch the game.

**Expected behavior**
The game launches normally.

**Log**
<https://paste.ee/p/UetX6>

This part looks important:

```
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Critical injection failure: @Inject annotation on onSlotUpdate could not find any targets matching 'Lnet/minecraft/class_1703;method_7610(Ljava/util/List;)V'
```

**Additional context**
As noted in the title, and as is apparent from the logs, this occurs when the MouseWheelie is only enabled mod.

**Comment by Siphalor** [comment]
You're using an outdated version of Mouse Wheelie.

The version you used was built for the Release Candidate of 1.17 and is thus compatible with 1.17.
During the snapshot cycle of 1.17.1 Mouse Wheelie broke and I released a version for the pre-release of 1.17.1 and onwards. That version will run reliably on Minecraft 1.17.1.

**Comment by mattfbacon** [comment]
Ah, alright. That release was not on Spigot when I opened this issue. Thanks for the help.
